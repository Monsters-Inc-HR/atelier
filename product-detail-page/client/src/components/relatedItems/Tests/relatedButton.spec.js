/**
 * @jest-environment jsdom
 * @jest-environment-options {"url": "https://jestjs.io/"}
 */

import React from 'react';
import { render, fireEvent } from '@testing-library/react';
import RelatedButton from '../relatedButton.jsx';

describe('Should render related button component', () => {
  test('renders Button component', () => {
    const compareMock = jest.fn();

    const { getByText } = render(<RelatedButton />);

    fireEvent.click(getByText('‚≠ê'));

    expect(compareMock).toHaveBeenCalled();
  })
})