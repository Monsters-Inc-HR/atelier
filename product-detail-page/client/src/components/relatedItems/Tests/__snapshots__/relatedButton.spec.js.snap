// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render related button component renders Button component 1`] = `
<div>
  <button
    class="related-compare-button"
  >
    ‚≠ê
  </button>
</div>
`;
